pmultiqc/modules/common/plots/id.py

```
@@ -0,0 +1,956 @@
+            """
+
+    elif delta_mass_type == "quantms_ppm":
+        plot_id = "delta_mass_ppm"
+        plot_title = "Delta Mass [ppm]"
+        plot_xlab = "Delta Mass [ppm]"
+        sub_section_order = 2
+        description_text = """
+            Delta Mass [ppm] calculated from mzTab.
+            """
+        help_text = """
+            Delta Mass [ppm] calculated from mzTab: ((𝑒𝑥𝑝𝑒𝑟𝑖𝑚𝑒𝑛𝑡𝑎𝑙 𝑚/𝑧 - 𝑡ℎ𝑒𝑜𝑟𝑒𝑡𝑖𝑐𝑎𝑙 𝑚/𝑧) / (𝑡ℎ𝑒𝑜𝑟𝑒𝑡𝑖𝑐𝑎𝑙 𝑚/𝑧)) × 10^6.
```

Ruff (0.14.0)

String contains ambiguous `𝑧` (MATHEMATICAL ITALIC SMALL Z). Did you mean `z` (LATIN SMALL LETTER Z)?

(RUF001)